mitodl/mitxonline
# Online-1100 Disable cart and price on course page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/mitxonline/issues/1100

#### What's this PR do?

- Disables checkout if e-commerce is disabled.
- Disables discounted prices on the course page if e-commerce is disabled.
- Sorts the imports in the changed files.


#### How should this be manually tested?
1.
Approve financial assistance for a request.
Make sure `FEATURE_ENABLE_UPGRADE_DIALOG ` is disabled.
Visit the course page and verify that the price card displays the total price.
2. 
Add a product to the cart when e-commerce is enabled.
Now disable the e-commerce flag.
Visit /cart/ directly or refresh the cart page.
The cart should be empty.
